esp/esdllib/esdl_def_helper.cpp

```diff
@@ -113,6 +113,58 @@ void EsdlDefinitionHelper::setTransformParams( EsdlXslTypeId xslId, IProperties
     parameters.setValue( xslId, params );
 }
 
+void removeEclHiddenStructs(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*[@ecl_hide='1']");
```

Comment:
Yes JLIB will treat both single and double quoted attributes equivalently.  I use single because the string is more readable without the escapes.